Source/Core/Common/Random.h

```diff
@@ -5,12 +5,37 @@
 #pragma once
 
 #include <cstddef>
+#include <memory>
 #include <type_traits>
 
 #include "Common/CommonTypes.h"
 
 namespace Common::Random
 {
+/// Cryptographically secure pseudo-random number generator, with explicit seed.
+class PRNG final
+{
+public:
+  explicit PRNG(u64 seed = 0) : PRNG(&seed, sizeof(u64)) {}
```

Comment:
The seed probably shouldn't have a default value, lest someone accidentally set the seed to 0 when they wanted a random seed.

Reply:
Oh yeah, we should remove the default value to encourage the use of Random::Generate instead.